st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -348,35 +426,35 @@ def encrypt_archive(archive_file_path, debug=False):
     encrypted_archive_output_file_path = archive_file_path + '.asc'
```

Comment:
GPG_KEY at line 422 needs to be overridden like you have done for GPG_KEY_FINGERPRINT as well?